Composer/packages/client/src/pages/botProject/adapters/ABSChannelSpeechModal.tsx

```diff
@@ -72,7 +72,7 @@ const ABSChannelSpeechModal = (props: Props) => {
               }
             }}
           >
-            {formatMessage('Enable Speech')}
+            {formatMessage('Enable speech')}
```

Comment:
@zxyanliu @beyackle In this case I think Speech should be capitalized as it is referring to the name of the Speech channel / service.

Reply:
Thanks for the correction.

Reply:
@garypretty I agree. Speech should be capitalized. Thanks!